crates/zkforge/src/runner.rs

```diff
@@ -201,6 +203,51 @@ impl<'a> ContractRunner<'a> {
         Ok(setup)
     }
 
+    /// Sets up the cheatcode address with an empty contract so `vm.func()` calls do not panic.
+    fn setup_cheatcode_address(&mut self) {
```

Comment:
why do we have it `here` and copy pasterd in `executor`?

Reply:
This is no longer needed, removing.